### PR TITLE
Add support for Django 2.0

### PR DESCRIPTION
**Pull request by maxpeterson**
Drop support for Django < 1.10

### CODE REVIEW

**Comment by maxpeterson** [comment]
@meshy @Ian-Foote would either of you care to review this.

**Review by meshy — APPROVED** [review]
Looks good to me. As a stretch-goal, it might be worth building against some more modern versions of Python, but that can wait for another PR.

**Comment by maxpeterson** [comment]
Thanks @meshy I am moving a project that uses this to to 2.2 so it may yet need more updates.

File: .travis.yml [hunk 1]
```diff
@@ -9,8 +9,13 @@ env:
   global:
     - DATABASE_URL='postgres://postgres@localhost/test_utils'
   matrix:
+    - DJANGO='django~=2.0.0' DRF='djangorestframework~=3.11.0'
     - DJANGO='django~=1.11.0' DRF='djangorestframework~=3.9.1'
-    - DJANGO='django~=1.8.0' DRF='djangorestframework~=3.6.0'
+    - DJANGO='django~=1.10.0' DRF='djangorestframework~=3.6.0'
```
Comment (LilyFirefly) [inline .travis.yml:14]: I would drop Django 1.10 while we're at it. It went out of support in 2017.

File: .travis.yml [hunk 2]
```diff
@@ -9,10 +9,20 @@ env:
...
-install:
+    - DJANGO='django~=1.10.0' DRF='djangorestframework~=3.6.0'
+matrix:
+  exclude:
+    - env: DJANGO='django~=2.0.0' DRF='djangorestframework~=3.11.0'
+      python: 2.7
+services:
+  - postgresql
+addons:
+  postgresql: "9.4"
```
Comment (LilyFirefly) [inline .travis.yml:22]: It might be good to use a higher version of postgres here.

File: .travis.yml (hunk 1 above)
Comment (LilyFirefly) [reply .travis.yml:14]: Once we have a release supporting 1.11 to 2.2, I think we should drop everything below 2.2 and support 2.2, 3.0 and 3.1.

**Pull request merged**